prog/target.go

```diff
@@ -68,6 +68,8 @@ type Target struct {
 	// The default ChoiceTable is used only by tests and utilities, so we initialize it lazily.
 	defaultOnce        sync.Once
 	defaultChoiceTable *ChoiceTable
+
+	Files []string
```

Comment:
I think this should be a `map[string][]string` because if I have dirname["/proc/*"] and dirname["/sys/*] I need 2 separate sets of files. So this needs to map the string in dirname to the set of strings.

Reply:
Oh, I see, you use strings.HasPrefix check later. But I think it's somewhat unflexible and unreliable, it's better to explicitly separate sets of files.